source/api-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _rm-rest-api:
+
+============================
+Relational Migrator REST API
+============================
+
+.. facet::
+   :name: genre 
+   :values: tutorial
+
+.. meta:: 
+   :description: Learn about the MongoDB Relational Migrator REST API and how to use it.
+   :keywords: data migration, rest api, open api
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The Relational Migrator REST API provides you with programmatic access 
```

Comment:
Consider rewrite into format "To (state goal), (state action)" if you can. Cheers.